gradle.properties

```diff
@@ -1,6 +1,7 @@
 # Package definitions
 GROUP=io.arrow-kt
-VERSION_NAME=0.10.3-SNAPSHOT
+VERSION_NAME=1.3.70-SNAPSHOT
```

Comment:
Shouldn't this be 1.3.60-SNAPSHOT, unless we're jumping versions directly?

Reply:
We start with this version though it will change very soon when sync is ready.

Reply:
Thanks for the review @i-walker!!! :+1:

Reply:
I removed my comment after I read the conv. in Slack. My bad :)

Reply:
Don't worry @i-walker , thanks!! :+1: